test_sets/Ideas_for_SDL_must_send_WARNINGS_success_true_to_mobile_app_if_HMI_respond_WARNINGS_at_least_to_one_component_of_RPC.txt

```diff
@@ -0,0 +1,29 @@
+./test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_UI_SetGlobalProperties_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_SetGlobalProperties_TTS_SetGlobalProperties_SUCCESS
+./test_scripts/API/DeleteCommand/ATF_DeleteCommand_UI_DeleteCommand_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_DeleteCommand_VR_DeleteCommand_SUCCESS
+./test_scripts/API/DeleteCommand/ATF_DeleteCommand_VR_DeleteCommand_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(
+success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_VR_DeleteCommand_UI_DeleteCommand_SUCCESS
+./test_scripts/API/AddCommand/ATF_AddCommand_UI_AddCommand_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_AddCommand_VR_AddCommand_SUCCESS
+./test_scripts/API/AddCommand/ATF_AddCommand_VR_AddCommand_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_VR_AddCommand_UI_AddCommand_SUCCESS
+./test_scripts/API/Alert/ATF_Alert_UI_Alert_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_Alert_TTS_Speak_SUCCESS
+./test_scripts/API/Alert/ATF_Alert_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_UI_Alert_SUCCESS
+./test_scripts/API/AlertManeuver/ATF_AlertManeuver_Navi_AlertManeuver_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_Navi_AlertManeuver_TTS_Speak_SUCCESS
+./test_scripts/API/AlertManeuver/ATF_AlertManeuver_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_Navi_AlertManeuve_SUCCESS
+./test_scripts/API/PerformInteraction/ATF_PerformInteraction_UI_PerformInteraction_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_PerformInteraction_VR_PerformInteraction_SUCCESS
+./test_scripts/API/PerformInteraction/ATF_PerformInteraction_VR_PerformInteraction_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_VR_PerformInteraction_UI_PerformInteraction_SUCCESS
+./test_scripts/API/PerformAudioPathThru/ATF_PerformAudioPathThru_UI_PerformAudioPathThru_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_PerformAudioPathThru_TTS_Speak_SUCCESS
+./test_scripts/API/PerformAudioPathThru/ATF_PerformAudioPathThru_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_UI_PerformAudioPathThru_SUCCESS
+
+
+*****Precondition: HMI does NOT respond to Interface.IsReady

```

Comment:
@vvvakulenko, I didn't see tests that are covering for
AlertManeuver: 
Navi.IsReady is not sent - For this interface please clarify should be checked because is not linked in Exceptions

Alert: TTS.IsReady is not sent.

PerformAudioPassThru: TTS.IsReady is not sent.